config/logging.yaml

```diff
@@ -0,0 +1,83 @@
+# enable logging output inline in rails console
+:console_inline: false
+# colorize log including level token based on severity
+:colorize: false
+# adds an extra line after each log message with information from which file:line and method
+# was this message logged
+:log_trace: false
+#do not log certain paths (example  /notices/get_new)
+:ignored_paths: []
+# configuration of all loggers used in app
+:loggers:
+  # this logger is parent of all other loggers, by default this is the only one having appender
+  # which is used by all other loggers
+  :root:
+    # only messages with severity >= this level from all loggers will be logged
+    # valid values:  debug, info, warn, error, fatal, unknown
+    :level: warn
```

Comment:
normally we have info level logs in production, which show requests at least, is that still the case?